src/FSoperations.py

```
@@ -0,0 +1,50 @@
+import os
+import shutil
+from pathlib import Path
+
+
+def copy_files(src: str, dst: str):
+    
+    if dir == None:
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)